### PR TITLE
🎨 Project color display implementation

### PR DESCRIPTION
**Pull request by heytherewill**
Ref #9

### CODE REVIEW

**Review by shantanuraj — COMMENTED** [review]
We should log time-entries with their projects in the `list` command as well?

**Comment by heytherewill** [comment]
Yeah, we will. This PR does not fetch any entities, that's why it's not visible yet. This is just groundwork

File: src/models.rs [hunk 1]
```diff
@@ -14,6 +18,124 @@ pub struct User {
...
+    pub static ref HAS_TRUECOLOR_SUPPORT: bool = if let Ok(truecolor) = env::var("COLORTERM") {
+        truecolor == "truecolor" || truecolor == "24bit"
+    } else {
+        false
+    };
+}
+
+impl Project {
+    /// Gets the closest plain color to the TrueColor
+    pub fn name_in_closest_terminal_color(&self, red: u8, green: u8, blue: u8) -> ColoredString {
```
[the remark this answers is not in the document]
Comment (heytherewill) [reply src/models.rs:57]: I don't think any other entity will need true colors, so should the need arise we can do so

**Review by shantanuraj — APPROVED** [review]
Let's go!

**Pull request merged**